sdcm/cluster.py

```diff
@@ -3611,6 +3611,26 @@ def verify_node_setup(start_time):
     return wrapper
 
 
+def get_partition_keys(ks_cf: str, db_node: BaseNode, pk_name: str = 'pk', num_of_pk: int = 100) -> List[str]:
+    """
+    Return list of partitions from a requested table
+    :param ks_cf:
+    :param db_node:
+    :param num_of_pk:
+    :param pk_name:
+    :return:
+    """
+    pks_list = []
+    cmd = f'select {pk_name} from {ks_cf} limit {num_of_pk}'
```

Comment:
Why with cqlsh and not with driver?
Please change

Reply:
@roydahan , why not? it is just a small "internal" query

Reply:
because everything that we can should be done through driver.

Reply:
fixed.